README.md

```diff
@@ -1,11 +1,23 @@
-# Open 3D Engine Networking Special Interest Group
+# Open 3D Engine Special Interest Group on Networking
 
-Materials for the Open 3D Engine Foundation Networking Special Interest Group (O3DE Network SIG).
-We manage the networking components including foundational pieces, transport, Multiplayer, client/server interactions and Cloud connectivity pieces of O3DE.
+This repoistory holds materials for the Open 3D Engine Foundation Special Interest Group for Networking (SIG-Network).
+
+We manage the networking components for O3DE including foundational networking layers, transport, Multiplayer, client/server interactions and Cloud connectivity pieces of O3DE. See the [SIG's charter](governance/SIG%20Network%20Charter.md) for further details.
+
+## How do I get involved?
+
+You can:
+* Participate in the SIG Chat channel, **#sig-network** on O3DE's [Discord Server](https://discord.com/invite/o3de)
+* Attend any of our public meetings by connecting to the **#sig-network** voice channel on O3DE's Discord server. See the [O3DE Calendar](https://lists.o3de.org/g/o3de-calendar/calendar) for upcoming meetings and details.
+    * The SIG hosts a general public meeting once a month, on the third Thursday of the month, to discuss work completed, items of note to the SIG, pending RFCs and other related items. Look for the meeting titled **SIG-Network Meeting** on O3DE calendar. [Meeting agendas](https://github.com/o3de/sig-network/issues?q=is%3Aopen+is%3Aissue+label%3Amtg-agenda) are posted in advanced as GitHub issues and anyone can raise issues for discussion.
+    * The SIG also meets once per week, on Thursdays, to review any new issues of interest to the SIG. Look for the meeting titled **SIG-Network Issue Triage** and see the [Triage Guide](TRIAGE_GUIDE.md) for more details of what happens at this meeting meeting.
```

Comment:
Suggestion: The SIG also meets once per week, on Thursdays, to review any new GitHub and community issues.